openpype/plugins/publish/collect_sequence_frame_data.py

```diff
@@ -0,0 +1,47 @@
+import pyblish.api
+import clique
+
+
+class CollectSequenceFrameData(pyblish.api.InstancePlugin):
+    """Collect Sequence Frame Data
+    """
+
+    order = pyblish.api.CollectorOrder + 0.2
+    label = "Collect Sequence Frame Data"
+    families = ["plate", "pointcache",
+                "vdbcache", "online",
+                "render"]
+    hosts = ["traypublisher"]
+
+    def process(self, instance):
+        frame_data = self.get_frame_data_from_repre_sequence(instance)
+        for key, value in frame_data.items():
+            if key not in instance.data:
+                instance.data[key] = value
+                self.log.debug(f"Frame range data {key} has been collected ")
```

Comment:
Since we are logging per key, would it be nice to also report the value?

```suggestion
                self.log.debug(f"Collected sequence frame range data '{key}': {value}")
```